### PR TITLE
bump version

### LINKED ISSUE #8: @conda-forge-admin, please add user @casperdcl
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by casperdcl**
- updated v1.6.1 (closes #6, closes #5, closes #4, closes #3)
- added user casperdcl (closes #9, fixes #8)
- deps: add humanize (fixes #6)

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered](https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

/CC @xhochy as requested in https://github.com/conda-forge/zenodo_get-feedstock/pull/9#issuecomment-2112615389

### CODE REVIEW

**Comment by casperdcl** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,46 +1,40 @@
...
   entry_points:
-    - zenodo_get = zenodo_get.zget:zenodo_get
+  - zenodo_get = zenodo_get.zget:zenodo_get
   script: {{ PYTHON }} -m pip install . -vv
-
 requirements:
   host:
-    - pip
-    - python >=3.6
+  - pip
```
Comment (xhochy) [inline recipe/meta.yaml]: Please restore the formatting here. The old indentation is how it should be.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (casperdcl) [reply recipe/meta.yaml]: Done

**Review by xhochy — APPROVED** [review]

**Comment by casperdcl** [comment]
Thanks @xhochy - any idea why it's not building?

**Comment by xhochy** [comment]
No idea, make an empty commit to main. It could be that this was due to some glitch in Github's notification system.

**Pull request merged**